test/web-platform-tests/to-upstream/html/browsers/browsing-the-web/navigating-across-documents/fragment-replace.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
This test doesn't pass in browsers. (Save it locally and add `https://w3c-test.org/` in front of the script URLs.)

I think this is because if you do multiple window.location.assign()s in a row, only the last one has any impact? You can confirm by logging (and asserting against) `window.history.length` in various places.